packages/medusa/src/models/line-item.ts

```diff
@@ -72,8 +92,8 @@ export class LineItem extends BaseEntity {
   @Column({ nullable: true })
   description: string
 
-  @Column({ nullable: true })
-  thumbnail: string
+  @Column({ type: "varchar", nullable: true })
+  thumbnail: string | null
```

Comment:
nit(non-blocking): As mention somewhere 😂 if you add null here (since the field is nullable) I think that for the consistency it should also be added on the other nullable column of the model on which the improvement is done

Reply:
Ok 😂 in that case don’t touch that here for now, but could you create an issue for enhancements referencing this? Because it could create some unexpected result since you think you get a string but in fact it is null